AutomatedNotebooks-Manager.ipynb

```diff
@@ -0,0 +1,327 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.            path = f"https://ml.azure.com/fileexplorerAzNB?wsid=/subscriptions/{AML_details['sub_id']}/resourcegroups/{AML_details['RG']}/workspaces/AzureMLWorkspace&amp;tid={AML_details['ten_id']}&amp;activeFilePath=Users/pebryan/{notebook_name}"
</pre><p>You can wrap really long strings like this - you can put a + between the lines but Python auto-concatenates adjacent strings</p><p>path = (</p><p>	"str1"</p><p>	"str2"</p><p>	...</p><p>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>